pkg/flare/archive.go

```diff
@@ -239,7 +240,7 @@ func getConfigFiles(fb flaretypes.FlareBuilder, confSearchPaths map[string]strin
 	}
 }
 
-func getProcessChecks(fb flaretypes.FlareBuilder, getAddressPort func() (url string, err error)) {
+func getProcessChecksFromProcessAgent(fb flaretypes.FlareBuilder, getAddressPort func() (url string, err error)) {
```

Comment:
How about just `getChecksFromProcessAgent`, no need to say "Process" twice.

Reply:
Done